spec/integration/post_index_spec.rb

```diff
@@ -0,0 +1,59 @@
+require 'rails_helper'
```

Comment:
- [ ] After fixing and running the tests, the tests still fail due to some selenium driver issue. Kindly resolve that.

  ![image](https://github.com/AlyAtalla/blog-app/assets/36542986/f2e5834f-1fa3-4420-b101-0c58979d1255)